### PR TITLE
[14.0][MIG] Module stock_picking_invoicing

### PR DESCRIPTION
**Pull request by mbcosta**
Migration module stock_picking_invoicing to v14, the PR https://github.com/OCA/account-invoicing/pull/1010 are missing the last commits made in v13:

image

cc @rvalyi @renatonlima @mileo @zar21

### CODE REVIEW

File: stock_picking_invoicing/wizards/stock_invoice_onshipping.py [hunk 1]
```diff
@@ -498,15 +498,15 @@ def _update_picking_invoice_status(self, pickings):
         return pickings._set_as_invoiced()
 
     def ungroup_moves(self, grouped_moves_list):
-        """ Ungrup your moves, split them again, grouping by
+        """Ungrup your moves, split them again, grouping by
```
Comment (hparfr) [inline stock_picking_invoicing/wizards/stock_invoice_onshipping.py:497]: Ungroup

Comment (hparfr) [inline stock_picking_invoicing/wizards/stock_invoice_onshipping.py]:
```suggestion
        """Ungroup your moves, split them again, grouping by
```

File: stock_picking_invoicing/wizards/stock_invoice_onshipping.py [hunk 2]
```diff
@@ -498,15 +498,15 @@ def _update_picking_invoice_status(self, pickings):
...
         fiscal position, max itens per invoice and etc
         :param grouped_moves_list:
         :return: list of grouped moves list
         """
         return [grouped_moves_list]
 
     def _create_invoice(self, invoice_values):
-        """ Overrite this metothod if you need to change any values of the
+        """Overrite this metothod if you need to change any values of the
```
Comment (hparfr) [inline stock_picking_invoicing/wizards/stock_invoice_onshipping.py]: Override

Comment (rvalyi) [reply stock_picking_invoicing/wizards/stock_invoice_onshipping.py]: metothod -> method

**Review by hildickethan — APPROVED** [review]

**Review by rvalyi — CHANGES_REQUESTED** [review]
Hello @mbcosta 

1. this commit seems an error: when removing such decorators, methods become @api.multi by default. Something there are not as they work on the RecordSet Model and not a recordset itself (look at other default_get declarations in other modules) https://github.com/OCA/account-invoicing/pull/1014/commits/f7662ba33dc86cbae0036b1f025806b222fd41cf
2. contributors are not expected to re-generate REAME files in specific commits such as https://github.com/OCA/account-invoicing/pull/1014/commits/6773f416d7fe48392f4ae70a32b252c677f3c2a9 as these are re-generated by the OCA bots anyway. So better if you remove that commit too.
3. Once this commit is removed it's better if you squash https://github.com/OCA/account-invoicing/pull/1014/commits/4fe918a17f9bad6e729c169158d8ccde4ec989da onto the previous migration commit because such trivial code formating commits don't deserve dedicated commit (it pollutes the log).

Use `rebase -i` for this cleanup. The othercommits are OK.

**Review by rvalyi — APPROVED** [review]

**Comment by mbcosta** [comment]
Thanks all for reviews, the PRs https://github.com/OCA/account-invoicing/pull/1010 and https://github.com/OCA/account-invoicing/pull/800 should be closed after this merge.

**Comment by rvalyi** [comment]
> Thanks all for reviews, the PRs #1010 and #800 should be closed after this merge.

Done!

**Pull request merged**